packages/oui-navbar/_mixins.less

```diff
@@ -6,136 +6,461 @@
-      display: inline-block;
+    @media screen and (max-width: 1024px) {
+      &[aria-haspopup="true"]::after {
+        content: "\F110";
+        font-family: 'Oui Icons';
+        color: @arrow-color;
+        padding: @padding;
+        position: absolute;
+        top: 50%;
+        right: 0;
+        transform: translateY(-50%);
+      }
+    }
+
+    .active > & { color: @font-color_active; }
+  }
+
+  .navbar-link-variant(
+    @font-size,
+    @font-color,
+    @background-color,
+    @background-color_hover,
+    @width: auto
+  ) {
+    width: @width;
+    font-size: @font-size;
+    color: @font-color;
+    background-color: @background-color;
+
+    &:hover,
+    &:focus {
+      color: @font-color;
+      background-color: @background-color_hover;
+    }
+  }
+
+  .navbar-link-icon(
+    @icon-size: @oui-navbar-icon-size,
+    @icon-spacing: @oui-navbar-icon-spacing
+  ) {
+    .oui-icon {
+      width: @icon-size;
+      height: @icon-size;
+      margin-right: @icon-spacing;
+      font-size: @icon-size;
+      color: inherit;
+      vertical-align: text-top;
+    }
+
+    &[aria-haspopup]::after {
+      display: none;
+    }
+
+    @media screen and (max-width: 1024px) {
       padding: 0;
-      font-size: rem-calc(14);
-      font-weight: 600;
-      text-decoration: none;
+      font-size: 0;
+      text-align: center;
+
+      .oui-icon {
+        margin: 0;
+        vertical-align: middle;
+      }
     }
+  }
 
-    &__title-text {
-      display: inline-block;
-      font-size: rem-calc(28);
-      line-height: 1;
+  .navbar-menu(
+    @height: @oui-navbar-height,
+    @menu-width: @oui-navbar-menu-width,
+    @menu-padding: @oui-navbar-menu-padding,
+    @menu-background-color: @oui-navbar-menu-background-color,
+    @menu-background-color_hover: @oui-navbar-menu-background-color_hover,
+    @menu-border-bottom: @oui-navbar-menu-border-bottom,
+    @menu-box-shadow: @oui-navbar-menu-box-shadow,
+    @menu-font-color: @oui-navbar-menu-font-color,
+    @menu-font-size: @oui-navbar-menu-font-size,
+    @menu-font-size_small: @oui-navbar-menu-font-size_small,
+    @menu-z-index: @oui-navbar-menu-z-index,
+    @menu-title-font-color: @oui-navbar-menu-title-font-color,
+    @menu-title-font-size: @oui-navbar-menu-title-font-size,
+    @menu-title-line-height: @oui-navbar-menu-title-line-height,
+    @menu-back-padding: @oui-navbar-menu-back-padding
+  ) {
+    width: 100%;
+    min-width: @menu-width;
+    position: absolute;
+    top: 100%;
+    left: 0;
+    color: @menu-font-color;
+    background-color: @menu-background-color;
+    box-shadow: @menu-box-shadow;
+    z-index: @menu-z-index;
+
+    @media screen and (max-width: 1024px) {
+      min-width: 0;
+      max-width: @menu-width;
+      box-shadow: none;
     }
 
-    &__title-bold {
-      display: inline-block;
-      margin-left: rem-calc(8);
-      font-size: rem-calc(28);
-      font-weight: 600;
+    @media screen and (min-width: 1025px) {
```

Comment:
I wonder if 1025 is an error or not...

Reply:
Not an error: min-width (for desktop only)